filament/backend/src/vulkan/VulkanHandles.h

```diff
@@ -147,13 +151,24 @@ struct VulkanDescriptorSet : public VulkanResource, HwDescriptorSet {
         }
     }
 
+    void setOffsets(backend::DescriptorSetOffsetArray&& offsets) noexcept {
+        mOffsets = std::move(offsets);
+    }
+
+    backend::DescriptorSetOffsetArray* getOffsets() {
```

Comment:
shouldn't this be `const* getOffsets() const`

Reply:
good catch. done